config/dev

```diff
@@ -1,5 +1,11 @@
+instanceid = dev
+
 include Makefile
 
+export base_url = /${instanceid}
```

Comment:
This is used by wsgi, it's a common way of all c2c projects to set instance name in path to allow many devs on the same server.